mathematica/ieee754_floating_point_evaluation_test.wl

```diff
@@ -296,6 +312,76 @@ End[]
 (*IEEEEvaluateWithRelativeError*)
 
 
+?IEEEEvaluateWithRelativeError
```

Comment:
Could we have tests that use `Exact[`…`]` instead of the private applyOpWithRelativeError?